src/actions/analytics.ts

```diff
@@ -180,19 +150,8 @@ export function trackAction(action: string, data: Record<string, unknown>) {
     (acc, key) => ({ ...acc, [key]: data[key] ? data[key] : '1' }),
     {},
   );
-  // Format event data to correct firebase format
-  const firebaseContextData = Object.entries(data).reduce(
-    (acc, [key, value]) => ({
-      ...acc,
-      [formatFirebaseEvent(key)]: value || '1',
-    }),
-    {},
-  );
-  // Format the event names to firebase format
-  const firebaseEventName = formatFirebaseEvent(action);
-  // Log event to firebase
-  FBAnalytics().logEvent(firebaseEventName, firebaseContextData);
-  return () => RNOmniture.trackAction(action, newData);
+
+  return () => FBAnalytics().logEvent(formatFirebaseEvent(action), newData);
```

Comment:
There are analytics events that are not within this file. I was debating whether it would be better to go all throughout the codebase and change the action names to align with the firebase convention, or if it would be simpler to just leave the format function in here. Let me know what you think.

Reply:
Might be nice to rename them? But that can be future work...